tests/test_container.py

```diff
@@ -1181,7 +1187,7 @@ def test_get_objects_stream_closes(temp_container, generate_random_data):
         obj_md5s.keys(), skip_if_missing=True
     ) as triplets:
         # I loop over the triplets, but I don't do anything
-        for _ in triplets:
+        for _ in triplets:  # pylint: disable=not-an-iterable
```

Comment:
I think pylint has been upgraded and causes these errors. I verified and the type ares are correct. There seems to be a issue related to this https://github.com/pylint-dev/pylint/issues/9252

I put these into a separate commit